packages/go_tools.rb

```diff
@@ -3,11 +3,11 @@
 class Go_tools < Package
   description 'Developer tools for the Go programming language'
   homepage 'https://github.com/golang/tools'
-  version '0.6.6'
+  version '2025.1.1'
```

Comment:
This is another case where anitya got the wrong version.  See https://github.com/golang/tools/releases.

Reply:
This is now handled, and the package is rebuilding.